frontend/src/components/alerts/Info.js

```diff
@@ -1,6 +1,6 @@
 const Info = props => (
   <div
-    class="mb-4 flex w-full items-center rounded-lg bg-blue-50 p-4 text-sm text-blue-800 dark:bg-gray-800 dark:text-blue-400"
```

Comment:
Hmm any reason to remove this mb? cause i remeber relying on the mb in other places to have space between elements

Reply:
ah yes, it was adding empty space within a container. I meant to add this margin back to wherever you used. I'll do that